packages/web-react/src/components/Flex/index.ts

```diff
@@ -1,5 +1,4 @@
 'use client';
 
-export * from './Flex';
-export * from './useFlexStyleProps';
```

Comment:
**note (if-minor):** When there are only [named exports](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export) the `*` asterisk is ok here. So when you add a new function and export it, you do not need to write it here like you do it in your modification. So I think for the hook it was ok.